README.md

```diff
@@ -2,24 +2,250 @@
+distributed computing system services.
 
-## Contributing to the PMIx Standard
+## Governance
 
-There are two main ways to contribute to the PMIx Standard: Participating in the discussion, and writing sections of the document.
+The PMIx Standard is governed by an Administrative Steering Committee (ASC) comprised of representatives from a consortium of research, academic, and industrial organizations.The ASC shall consist of one representative from each Member organization, as defined below, and shall perform the following functions:
 
-### Participating in the discussion
+* Define the mission of the PMIx Standard.
+* Plan release timelines for versions of the PMIx Standard based on available resources, target features, and target goals of the Member organizations.
+* Vote on all issues requiring community consensus (e.g., acceptance of proposed modifications to the Standard), one vote per Member organization. In addition to voting on proposed modifications, the ASC can be requested to vote on any issue submitted by a Member for consideration.
+* Assemble working groups to address specific areas of interest to the community.
+* Nominate and approve the appointment of individuals to fill the role of Release Manager (per release), as described below.
 
-If you have a question about the PMIx interface or want to raise a question for discussion then please file an Issue against this repository. We are using the Issues to discuss topics, and will link Pull Requests related to that topic to that Issue. This provides the community with a public, archived forum for discussion.
+The ASC is led by two Co-Chairs responsible for organizing and conducting the quarterly meetings. This includes:
 
-### Participatin in the writing
+* Announcing meeting agendas at least one month in advance of the meeting. Any proposal wishing to be considered for the next meeting must be put forward (by assigning the proper label, as described below) prior to that time. The ASC Co-Chairs may allow a proposal to be added to the agenda after this time on a case-by-case basis at their sole discretion.
+* Chairing the quarterly meetings and ensuring an opportunity for all to express their opinion during the meeting
+* Maintaining and publishing minutes of the meeting, and tallying votes.
 
-If you want to write a portion of the standard (or even fix a typo or two), then use the Pull Request mechanism to post the change for review.
+Co-Chairs may vote on behalf of their organization subject to the one vote per Member organization limit. Co-Chairs serve for a period of two years, with seats filled on alternate years - i.e., one seat will be elected during the first meeting of each even year, and the other seat will be elected during the first meeting of each odd year. There is no restriction on the number of consecutive terms a given individual may serve - however, it is expected and desirable that the positions rotate amongst the Members.
```

Comment:
What is the election process for the chairs? Meaning, is it like the voting rules described below or is it different?

Reply:
> Co-Chairs are elected from amongst the ASC members during the first meeting of each calendar year (using the voting system described below) and serve for a period of two years, with seats filled on alternate years - i.e., one seat will be elected during the first meeting of each even year, and the other seat will be elected during the first meeting of each odd year.